### PR TITLE
Support for helm packages with deis.

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: roles/helm/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+helm_url: " https://get.helm.sh "
```
Comment (tayzlor) [inline roles/helm/defaults/main.yml]: extra spaces here between the "

File: roles/helm/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    creates: "{{ helm_bin }}"
+  environment: "{{ proxy_env }}"
+  tags:
+    - helm
+
+- name: install helm packages
+  become: yes
+  become_user: root
+  run_once: True
+  when: "helm_{{ item.name }}_enabled | bool"
```
Comment (tayzlor) [inline roles/helm/tasks/main.yml]: wondering if it makes sense to use this pattern here? We could just provide a list of packages to install. By default that could include Core packages or be empty and if populated would run the install. Then it wouldn't be necessary to provide a package list and a toggle variable. I'm not sure if that makes it better or worse though.

We don't provide a method for uninstalling a package either here.

File: roles/helm/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+- name: install helm
+  become: yes
+  become_user: root
+  command: "curl -s {{ helm_url }} | bash"
```
Comment (wallies) [inline roles/helm/tasks/main.yml]: can we do this a safer way?

File: roles/helm/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  args:
+    chdir: "{{ helm_folder }}"l
```
Comment (tayzlor) [inline roles/helm/tasks/main.yml]: build failure because of ```l``` at end of line here

**Comment by tayzlor** [comment]
Whats the crossover here https://github.com/kubernetes/helm

File: roles/helm/tasks/main.yml (hunk 2 above)
Comment (enxebre) [inline roles/helm/tasks/main.yml]: mm we can ensure package is not installed when helm_{{ item.name }}_enabled=false
Not sure what you mean in the first bit, I like toggle thing so people can ensure the state if their cluster by overriding defaults with env variables.

**Comment by wallies** [comment]
@tayzlor if you look down the bottom of that repo you will see 'Relationship to Google Cloud Platform's Deployment Manager and Deis's Helm'. It would seem they have merged into kubernetes helm.

**Comment by tayzlor** [comment]
Created https://github.com/Capgemini/kubeform/issues/71 as a follow-up for some docs

**Pull request merged**